src/library/payfine/PayFineUI.java

```diff
@@ -3,7 +3,7 @@
 
 public class PayFineUI {
     
-    public static enum UI_STATE { INITIALISED, READY, PAYING, COMPLETED, CANCELLED };
+    public static enum UI_STATE {INITIALISED, READY, PAYING, COMPLETED, CANCELLED};

```

Comment:
Enum name needs to lose the underscore. Probably also the all caps.

Reply:
Resolution: https://github.com/9Mcgusto6/ITC205-Team17/pull/4/commits/ce7116224681e5a19ce7340aa9c48f6dc260fac0